src/cli.js

```diff
@@ -196,9 +196,19 @@ function loadIgnores(params) {
   if (!file && !params.exclude) {
     return [];
   }
+  if (file)
+    file = fs.realpathSync(file);
```

Comment:
is this common? Sorry if it is, I just haven't seen other node build tools use realpathSync before.